dissect/ntfs/secure.py

```diff
@@ -2,6 +2,7 @@
 
 import io
 from typing import BinaryIO, Iterator
+from functools import lru_cache
```

Comment:
```suggestion
import io
from functools import lru_cache
from typing import BinaryIO, Iterator
```
This should fix the linter.